src/drive/web/modules/navigation/Breadcrumb/MobileAwareBreadcrumb.spec.jsx

```diff
@@ -4,9 +4,10 @@ import { createMockClient } from 'cozy-client'
 import AppLike from 'test/components/AppLike'
 
 import { MobileBreadcrumb } from './MobileAwareBreadcrumb'
+import { dummyBreadcrumbPath } from 'test/dummies/dummyBreadcrumbPath'
 
 describe('MobileAwareBreadcrumb', () => {
-  it('works', async () => {
+  it('works on mobile', async () => {
```

Comment:
@Ldoppea : maybe I should precise here, that in mobile mode, we used the old Breadcrumb. What do you think about it?

Reply:
Seems like the describe is explicit enough. But I don't really like tests named with just "works". They are not descriptive enough. So your new name improves it a bit, let's keep it.